crates/runestick-macros/src/context.rs

```diff
@@ -489,7 +489,7 @@ impl Context {
         name: &TokenStream,
         install_with: &TokenStream,
         tokens: &Tokens,
-    ) -> Result<TokenStream, Vec<syn::Error>>
+    ) -> std::result::Result<TokenStream, Vec<syn::Error>>
```

Comment:
So you don't have to change these ones, because they're part of the proc macro. We only need to change the ones which are part of a macro expansion. This usually just means anything that's in a call to `quote!` or `quote_in!`.

Reply:
Okay I change that